ArcadiaParties-SPA/src/app/user/user-info/user-info.component.scss

```diff
@@ -0,0 +1,32 @@
+:host {
+  display: flex;
+  flex: 1 0 auto;
+  justify-content: center;
+  align-items: center;
+}
+
+.center {
+  text-align:center;
+}
+
+mat-card {
+  margin: 10% auto;
+  border-radius: 8px;
+}
+
+p {
+  font-size: 15pt;
+}
+
+mat-card-content {
```

Comment:
I suppose that you don't need to override styles for mat components. You had to, because you used it incorrectly, now all should be allright.

Reply:
It deleted